usr.bin/ssh/sshd.c

```diff
@@ -1000,6 +1008,15 @@ server_accept_loop(int *sock_in, int *sock_out, int *newsock, int *config_s,
 				child_close(&(children[i]), 0, 0);
 				break;
 			case 1:
+				if (children[i].config) {
+					error_f("startup pipe %d (fd=%d)"
+					    "early read", i, children[i].pipefd);
+					if (children[i].early)
+						listening--;
+					srclimit_done(children[i].pipefd);
+					child_close(&(children[i]), 0, 0);
```

Comment:
maybe even kill the child here? I don't know...

Reply:
i've added a kill SIGTERM